test/parallel/test-fs-cp.mjs

```diff
@@ -248,6 +248,33 @@ function nextdir(dirname) {
   );
 }
 
+// It allows copying when is not a directory
```

Comment:
mh.... I am really not sure about this comment, could you clarify it? :pray: 

maybe something like
```suggestion
// It allows copying symlinks in src to locations in dest with existing synlinks not pointing to a directory.
```
?

Reply:
sgtm!